calculator.rb

```diff
@@ -0,0 +1,90 @@
+# Calculator (Dinner) - Loraine Kanervisto
+
+# Starts with user prompts and data collection
+
+#Collects first number
+puts "You find a calculator on the ground. You pick it up. The screen says:\n
+Enter a number>>"
+first_num = gets.chomp.to_f
+
+#Checks first number
+while first_num != "0" && first_num.to_i == 0
```

Comment:
Hm, looks like I'd need to tweak the numerical data based on the operation. Especially if I wanted to accommodate decimal places, 4.2/6